packages/react/src/PageHeader/PageHeader.tsx

```diff
@@ -435,6 +395,9 @@ const Breadcrumbs: React.FC<React.PropsWithChildren<ChildrenPropTypes>> = ({
             return value ? 'none' : 'flex'
           }),
           alignItems: 'center',
+          fontWeight: 'initial',
+          lineHeight: 'var(--text-body-lineHeight-large, 1.5)',
```

Comment:
Shouldn't we be using `var(--text-body-lineHeight-medium, 1.4285)`?

Reply:
Oh sorry I missed to update that! Just pushed it.